Agents.md

```
@@ -0,0 +1,208 @@
+
+```
+/
+├── src/
+│   ├── bot.ts                    # Entry point: initializes DB, bot, registers routes
+│   ├── config/
+│   │   └── db.ts                 # Mongoose connection helper
+│   ├── controllers/
+│   │   └── botController.ts      # Main router: registers all Telegram command/callback handlers
+│   ├── services/
+│   │   ├── inputService.ts       # Prompts user for text, price, and media input via chat
+│   │   ├── postService.ts        # Post formatting, preview, sending to groups, bump/sold editing
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)